descriptions/api.github.com/api.github.com.yaml

```diff
@@ -13661,37 +13473,6 @@ paths:
         category: actions
         subcategory: workflow-runs
       deprecated: true
-  "/repos/{owner}/{repo}/actions/runs/{run_id}/retry":
-    post:
-      summary: Retry a workflow
-      description: Retry your workflow run using its `id`. You must authenticate using
-        an access token with the `repo` scope to use this endpoint. GitHub Apps must
-        have the `actions:write` permission to use this endpoint.
-      tags:
-      - actions
-      operationId: actions/retry-workflow
-      externalDocs:
-        description: API method documentation
-        url: https://docs.github.com/rest/reference/actions#retry-a-workflow
-      parameters:
-      - "$ref": "#/components/parameters/owner"
-      - "$ref": "#/components/parameters/repo"
-      - "$ref": "#/components/parameters/run-id"
-      responses:
-        '201':
-          description: Response
-          content:
-            application/json:
-              schema:
-                type: object
-                properties: {}
-                additionalProperties: false
-      x-github:
-        githubCloudOnly: false
-        enabledForGitHubApps: false
-        previews: []
-        category: actions
-        subcategory: workflow-runs
```

Comment:
Is this endpoint defunkt? Can we deprecate it with `deprecationDate`/`removalDate` set instead of removing it?

Reply:
Looks like this change is correct, this endpoint was added as an experiment now has now been removed